### PR TITLE
drop support for Python 3.8

### PR DESCRIPTION
**Pull request by zacharyburnett**
Numpy is imminently dropping support for Python 3.8

### CODE REVIEW

**Comment by braingram** [comment]
Similar to stcal roman (which supports 3.8) tests against the development version of stpipe.
https://github.com/spacetelescope/stcal/pull/162#issuecomment-1508884573

File: .readthedocs.yaml [hunk 1]
```diff
@@ -16,9 +16,13 @@ formats:
   - htmlzip
   - pdf
 
+build:
```
Comment (WilliamJamieson) [inline .readthedocs.yaml:19]: Can you convert this to using mamba like astropy uses.

**Review by WilliamJamieson — CHANGES_REQUESTED** [review]

File: .readthedocs.yaml (hunk 1 above)
Comment (zacharyburnett) [reply .readthedocs.yaml:19]: Sure, but that's probably for another PR as it would also require an environment file

Comment (WilliamJamieson) [reply .readthedocs.yaml:19]: Well the environment you targeted in this PR is causing the docs to fail.

**Comment by zacharyburnett** [comment]
blocked by #94

**Review by WilliamJamieson — APPROVED** [review]

**Pull request merged**